### PR TITLE
Add ceph-docker-flake8

### PR DESCRIPTION
**Pull request by zmc**
This is a blatant ripoff of ceph-docker-lint, only for Python files.

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
Correctly failed this job: https://jenkins.ceph.com/job/ceph-docker-flake8/1/console

File: ceph-docker-flake8/build/build [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+function generate_filelist(){
+   if [[ "$pull_request_id" -eq "" || "${ghprbCommentBody:-}" = "jenkins flake8 all" ]]
+   then
+       find . -name '*.py'
+   else
+       curl -XGET "https://api.github.com/repos/ceph/ceph-docker/pulls/$pull_request_id/files" |
+       jq '.[].filename' |  # just the files please
+       tr -d '"' |  # remove the quoting from JSON
+       grep ".py$" # just the bash
```
Comment (GregMeno) [inline ceph-docker-flake8/build/build]: update comment to say "just the python"

**Comment by zmc** [comment]
@gmeno repushed

**Comment by zmc** [comment]
@GregMeno repushed

**Pull request merged**